cluster/addons/fluentd-elasticsearch/fluentd-es-image/Makefile

```diff
@@ -1,9 +1,11 @@
 .PHONY:	build push
 
+IMAGE = fluentd-elasticsearch
 TAG = 1.3
 
 build:	
-	sudo docker build -t kubernetes/fluentd-elasticsearch:$(TAG) .
+	docker build -t gcr.io/google-containers/$(IMAGE):$(TAG) .
+	docker tag -f gcr.io/google-containers/$(IMAGE):$(TAG) gcr.io/google-containers/$(IMAGE):$(TAG)
 
 push:	
-	sudo docker push kubernetes/fluentd-elasticsearch:$(TAG)
+	gcloud preview docker push gcr.io/google-containers/$(IMAGE):$(TAG)
```

Comment:
You should specify the project here so it doesn't need to be set in the environment: `gcloud --project=google-containers preview docker push ...`